src/borg/platform/windows.pyx

```diff
@@ -0,0 +1,80 @@
+import os
+import platform
+from functools import lru_cache
+
+
+cdef extern from 'windows.h':
+    ctypedef void* HANDLE
+    ctypedef int BOOL
+    ctypedef unsigned long DWORD
+
+    BOOL CloseHandle(HANDLE hObject)
+    HANDLE OpenProcess(DWORD dwDesiredAccess, BOOL bInheritHandle, DWORD dbProcessId)
+
+    cdef extern int PROCESS_QUERY_INFORMATION
+
+
+@lru_cache(maxsize=None)
+def uid2user(uid, default=None):
+    print(uid, default)
+    return default
+    # try:
+    #     return pwd.getpwuid(uid).pw_name
+    # except KeyError:
+    #     return default
```

Comment:
this should be cleaned up somehow before merge (remove print, remove commented code).

Reply:
(same also below)